tree/binary-tree-node.js

```diff
@@ -0,0 +1,113 @@
+class BinaryTreeNode {
+  constructor(value = null) {
+    this.value = value
+    this.left = null
+    this.right = null
+    this.parent = null
```

Comment:
Actually, it's still possible to implement a BST without referencing from child to parent.